charts/prometheus-redis-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v1.61.0
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 6.3.0
+version: 6.3.1
```

Features should be bumped to minor version.
```suggestion
version: 6.4.0
```